### PR TITLE
Added-mobile-toggle-menu

### PR DESCRIPTION
**Pull request by kinginthenorthcodez**
#Mobile Toggle menu
✔️ Added When the user clicks (or taps) the hamburger button, the mobile menu appears.
✔️Added When the user clicks (or taps) the close (X) button, the mobile menu disappears.
✔️Added When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
✔️Added  When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -21,17 +21,22 @@
...
+            <span></span>
+            <span></span>
+          </div>
+          <div class="navigation-menu">
+            <ul>
+              <li><a href="#portfolio">Portfolio</a></li>
+              <li><a href="#about">About</a></li>
+              <li><a href="#contact">Contact</a></li>
+            </ul>
+          </div>
```
Comment (descholar-ceo) [inline index.html:38]: - [ ] Just two concerns that need to be addressed before moving on:
  - When the first loads, the menu window is being rendered, <= it should not be like that, it should be the content of the page loaded instead,
  - When clicking on the contact menu, the page should scroll down to the `Contact me` section, kindly address these comments and submit another review request. 

At your best, please watch the following screencast for more context

https://user-images.githubusercontent.com/56886064/150879544-23b1fd1b-91f6-4a69-a1c7-3d108a4e388e.mp4

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
Hi @joshuaivie && @kinginthenorthcodez , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (joshuaoivie) [reply index.html:38]: Thank you for your review @descholar-ceo. The issues you raised have now been fixed.

**Review by Cathella — APPROVED** [review]
## 🔥 Status: Approved 🟢 🎉

### Hi @kinginthenorthcodez 👋🏻, 

You have done an awesome job making the requested changes 💪🏻!
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**